docs/packages/pkg/soil.rst

```diff
@@ -0,0 +1,21 @@
+.. spelling::
+
+    soil
+
+.. index:: 
+  single: graphics ; soil
+
+.. _pkg.soil:
+
+soil
+====
+
+-  https://www.lonesock.net/soil.html
+-  `Official <https://github.com/frequem/Simple-OpenGL-Image-Library>`__
+-  `Example <https://github.com/frequem/hunter/blob/master/examples/soil/CMakeLists.txt>`__
```

Comment:
Since you're targeting https://github.com/ruslo/hunter it should be `ruslo/hunter` instead of `frequem/hunter`.